src/main/java/org/broadinstitute/hellbender/tools/exome/ModeledSegment.java

```diff
@@ -26,14 +26,28 @@ public ModeledSegment(final SimpleInterval interval, final long targetCount, fin
         this(interval, NO_CALL, targetCount, segmentMean);
     }
 
+    /**
+     *  Get segment mean in log2 space
```

Comment:
not in this diff, but i noticed that the constructor in this class takes parameter `segmentMeanInLogCR`.  maybe rename to `segmentMeanInLog2CR` to be absolutely clear?